### ISSUE TITLE
Rule proposal: No zero fractions

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/eslint-plugin-unicorn#207]
There are no difference between `1` and `1.0` in JS. For consistency, the former should be preferred.

## Fail

```js
console.log(1.0 + 5.0);
console.log(3.00000000);
```

## Pass

```js
console.log(1 + 5);
console.log(3);
console.log(7.0000001);
```

**Comment by sindresorhus** [comment]
Accepted. PR welcome :)

**Comment by IssuehuntBot** [comment]
@issuehuntfest has funded $60.00 to this issue. [See it on IssueHunt](https://issuehunt.io/repos/55832243/issues/207)

**Comment by arshadkazmi42** [comment]
@sindresorhus I would like to take this.

**Comment by gurungrahul2** [comment]
@arshadkazmi42  I have already started the work :)

**Comment by arshadkazmi42** [comment]
@gurungrahul2 ok. thanks for information 👍

**Comment by stroncium** [comment]
Sorry about triple reference, didn't know github added cross-repo referencing, consider only the last one.

**Comment by IssuehuntBot** [comment]
@sindresorhus has rewarded $54.00 to @stroncium. [See it on IssueHunt](https://issuehunt.io/repos/55832243/issues/207)

- :moneybag: Total deposit: $60.00
- :tada: Repository reward(0%): $0.00
- :wrench: Service fee(10%): $6.00
